evalprompt/genSpecFromApi/prompts.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+const path = require('path');
+const { readFileSync } = require('fs');
+module.exports = async function (context) {
+    const srcPath = path.join(__dirname, 'src', `${context.vars.file}`);
+    const contexts = readFileSync(srcPath, 'utf8');
```

Comment:
`contexts`: better naming to `apiCode` or `code`?

Reply:
this is defined by promptfoo.